tests/e2e/e2e_setup_test.go

```diff
@@ -308,7 +308,8 @@ func (s *IntegrationTestSuite) initValidatorConfigs(c *chain) {
 		vpr.SetConfigFile(tmCfgPath)
 		s.Require().NoError(vpr.ReadInConfig())
 
-		valConfig := &tmconfig.Config{}
```

Comment:
This caused a panic with the new Tendermint version.

Reply:
I wonder if instead we want to use `TestConfig()`?

Reply:
Actually maybe since it's e2e we don't want to... @alexanderbez wdyt?

Reply:
You should use `DefaultConfig` as you do here. Why does it cause a panic? What is the panic?

In the SDK, you can see we use `DefaultConfig`: https://github.com/cosmos/cosmos-sdk/blob/main/testutil/network/network.go#L320-L322

Which follows down to here: https://github.com/cosmos/cosmos-sdk/blob/main/server/util.go#L60